HISTORY.rst

```diff
@@ -3,6 +3,14 @@
 Release History
 ===============
 
+0.12.2
++++++++++++++++
+
+**IoT Central updates**
+
+* Added missing "update" sub-commands for all commands supporting it 
+  ("device", "device-template", "file-upload-config", "organization", "user")
```

Comment:
personally I like having the full commands like so:
 * az iot central device
 * az iot central device-template

Reply:
done